File: godotopenxrpico/src/main/java/org/godotengine/openxrloaders/pico/GodotOpenXRPico.java
package org.godotengine.openxrloaders.pico;

import org.godotengine.godot.Godot;
import org.godotengine.godot.plugin.GodotPlugin;

/**
 * Godot plugin for the Pico OpenXR loader.
 */
public class GodotOpenXRPico extends GodotPlugin {

    public GodotOpenXRPico(Godot godot) {
        super(godot);
    }

    @Override
    public String getPluginName() {
        return "GodotOpenXRPico";
    }
}

File: godotopenxrpico/src/main/java/org/godotengine/openxr/vendors/pico/GodotOpenXRPico.kt
package org.godotengine.openxr.vendors.pico

import org.godotengine.godot.Godot
import org.godotengine.godot.plugin.GodotPlugin

/**
 * Godot plugin for the Pico OpenXR loader.
 */
class GodotOpenXRPico(godot: Godot?) : GodotPlugin(godot) {
    override fun getPluginName(): String {
        return "GodotOpenXRPico"
    }
}
